plugin.rb

```diff
@@ -78,6 +78,10 @@ def response(env)
 # We'll store the value in the user associated account's extra attribute hash using the full path as the key.
 DiscoursePluginRegistry.define_filtered_register :oauth2_basic_additional_json_paths
 
+# After authentication, we'll use this to confirm that the registered json paths are fulfilled,
+# or display an error message to the user.
+DiscoursePluginRegistry.define_filtered_register :oauth2_basic_required_json_paths
+
```

Comment:
Maybe worth adding a tiny bit of documentation in the comment so people know what keys need to be passed to the API?